creator-node/package.json

```diff
@@ -19,7 +19,7 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@audius/libs": "1.2.93",
+    "@audius/libs": "1.2.102",
     "JSONStream": "^1.3.5",
     "axios": "^0.19.2",
     "base64-url": "^2.3.3",
```

Comment:
lol so `ipfs-http-client` package is only used to import `Buffer`, idk why but i don't think we need it?

`ipfs-http-client-latest` is only used for CID import, which maybe we can also look into importing directly or something

either way, i dont think we should increase scope of this PR any further, can you add a follow up card to do that?

Reply:
card to investigate: https://linear.app/audius/issue/ASI-1029/remove-final-ipfs-leftovers